asv_bench/benchmarks/groupby.py

```diff
@@ -493,6 +493,39 @@ def time_frame_agg(self, dtype, method):
         self.df.groupby("key").agg(method)
 
 
+class CumminMax:
+    param_names = ["dtype", "method"]
+    params = [
+        ["float64", "int64", "Float64", "Int64"],
+        ["cummin", "cummax"],
+    ]
+
+    def setup(self, dtype, method):
+        N = 500_000
+        rg = np.random.default_rng(seed=0)
+        vals = rg.integers(
+            0,
+            1000,
```

Comment:
is there a reason you are doing things completely different that the setup for other existing groupby benchmarks?

Reply:
Can revert this change (this was a relic from when the benchmark was larger and was trying to cut down setup time). The new `numpy` random generator is slightly more efficient, but pretty negligible compared to other setup costs.